robottelo.properties.sample

```diff
@@ -79,3 +79,44 @@ driver=firefox
 # Default set to be 0, i.e. no timing of performance is measured and thus no
 # interference to original robottelo tests.
 test.foreman.perf=0
+
+# Folowing entries are used for preparation of performance tests after a fresh
+# install. They will be used by
+# `test/foreman/performance/test_standard_prep.py`, which supports:
+#
+# 1. downloading manifest,
+# 2. uploading manifest to subscription,
+# 3. updating Red Hat CDN URL,
+# 4. enabling key repositories: rhel6-rpms, rhel7-rpms, rhel6-kickstart-rpms,
+#    rhel7-kickstart-rpms, rhel6-optional-rpms, rhel7-optional-rpms,
+#    rhel6-optional-source-rpms, rhel7-optional-source-rpms,
+#    rhel6-optional-debug-rpms, r7-optional-debug-rpms
+#
+# Note that this preparation step is not required as long as satellite server
+# is already configured.
+test.cdn.address=http://example.redhat.cdn/pub
+
+# Set variables for a list of virtual machines' IP address, which would conduct
+# concurrent system subscriptions. Assume that those vms are pre-provisioned.
+# As for current workaround, append each IP address to the end, join by ","
+# without any spaces.
```

Comment:
Is this correct?

> A list of VM IP addresses or hostnames. Each system should already be provisioned. They will be used in concurrent system subscription tests.

I'm simply trying to find a more natural way to explain what's going on here.